src/camvidlog2/moondream/cli.py

```
@@ -0,0 +1,88 @@
+
+from camvidlog2.common.data import load as data_load
+from camvidlog2.moondream.ai import generate_bboxes
+from camvidlog2.moondream.data import create
+from camvidlog2.moondream.plot import overlay_detections
+from camvidlog2.vid import generate_frames_cv2, get_video_stats, save_video
+
+app = typer.Typer()
+
+
+@app.command()
+def load(
```

Ruff (0.12.2)

Missing return type annotation for public function `load`

Add return type annotation: `None`

(ANN201)